exp/services/ledgerexporter/internal/config.go

```diff
@@ -16,20 +17,31 @@ import (
 	"github.com/stellar/go/support/datastore"
 	"github.com/stellar/go/support/errors"
 	"github.com/stellar/go/support/ordered"
+	"github.com/stellar/go/support/storage"
 )
 
-const Pubnet = "pubnet"
-const Testnet = "testnet"
+const (
+	Pubnet    = "pubnet"
+	Testnet   = "testnet"
+	UserAgent = "ledgerexporter"
+)
+
+type Mode int
+
+const (
+	ScanFill Mode = iota
```

Comment:
```suggestion
	_ Mode = iota
	ScanFill Mode = iota
```

it's better to start at 1 so that the default value (0) can be invalid. This will allow us to distinguish the case where a Mode was explicitly set to `ScanFill` vs being unassigned completely

Reply:
yes, that is a good convention to follow, done - https://github.com/stellar/go/pull/5335/commits/d4aada61ec331c11741e7e1eae931ebe13e0f1db